src/adapters/PeopleAdapter.js

```diff
@@ -7,14 +7,33 @@ import {throwError} from 'rxjs';
  */
 export default class PeopleAdapter {
   /**
-   * Returns an observable that emits person data
+   * The status a person can have.
+   * @typedef {null|'active'|'bot'|'call'|'dnd'|'group'|'inactive'|'meeting'|'ooo'|'presenting'|'self'|'typing'} PersonStatus                      a              a      a    a
+   */
+
+  /**
+   * A Person object with details about the person.
+   * @typedef {Object} Person
+   * @property {string} ID The person identifier.
```

Comment:
Not sure how this would affect doc generation, but isn't it a bit more readable to have the descriptions aligned? That's what I did for [activity on my PR](https://github.com/webex/components/blob/3d72bc1965f192ac60b2e6bc3ca1ba9fc2ebf372/src/adapters/ActivityAdapter.js#L13), and how they have it in JSDocs example, https://jsdoc.app/tags-property.html

Reply:
Sure!